targets/ath79-generic

```diff
@@ -73,6 +73,11 @@ device('ocedo-raccoon', 'ocedo_raccoon', {
 
 -- TP-Link
 
+device('tp-link-archer-c2-v3', 'tplink_archer-c2-v3', {
+	packages = ATH10K_PACKAGES_QCA9887,
+  class = 'tiny',  -- 64M ath9k + ath10k
```

Comment:
Well, the indentation looks off here. And in addition to tiny, it should also be marked as broken. 

Missed that in the first round. sorry.

Reply:
> 64M RAM, guess we'll mark it as BROKEN.

I said as much in the OP.